### DISCUSSION TITLE
[PSD5] Exercise 2.4

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#141] [Esercizi (A-L)]
![image](https://user-images.githubusercontent.com/69389461/210015138-6534e1d9-dc7a-4f4c-8a89-297d233c92f4.png)

- _**N.B.:** Le tavole della verità sono le stesse dell'Esercizio 2.2 (#139 ), ma questa volta l'equazione booleana va scritta in forma canonica product-of-sums (POS)._

**Comment by Elia-Belli** [comment]
**Forma Canonica POS**: non minimizzata
(a) $\prod{(0)} = (A+B)$

(b) $\prod{(0,5,7)}=(A + B + C)(\bar A + B + \bar C)(\bar A + \bar B + \bar C)$

(c) $\prod{(0,2,3,4,5)}= (A+B+C)(A + \bar B + C)(A + \bar B + \bar C)(\bar A + B + C)(\bar A + B + \bar C)$

(d) $\prod{(1,4,5,9,11,12,13,14,15)}= (A +B+C + \bar D)(A + \bar B + C + D)(A + \bar B + C + \bar D)(\bar A + B + C + \bar D)$
$(\bar A + B + \bar C + \bar D)(\bar A + \bar B + C +D)(\bar A + \bar B + C + \bar D)(\bar A + \bar B + \bar C + D)(\bar A + \bar B + \bar C + \bar D)$

(e) $\prod{(0,1,2,4,5,12,13,14,15)}= (A+B+C+D)(A+B+C+ \bar D)(A+B+\bar C + D)(A+ \bar B + C + D)$
$( A + \bar B + C + \bar D)(\bar A + \bar B + C + D)(\bar A + \bar B + C + \bar D)(\bar A + \bar B + \bar C + D)(\bar A + \bar B + \bar C + \bar D)$

**Comment by FeddyLix17** [comment]
@Elia-Belli 
(c) tutto corretto tranne m<sub>3</sub> ( è (A+B&#x0304;+C&#x0304;) invece di (A&#x0304;+B&#x0304;+C&#x0304;) ) mancano m<sub>4</sub> e m<sub>5</sub> che sono rispettivamente (A&#x0304;+B+C) e (A&#x0304;+B+C&#x0304;)
per il resto tutto corretto

**Comment by Elia-Belli** [comment]
@FedVlogger17 corretto, grazie
